website/netlify.toml

```diff
@@ -12,3 +11,9 @@
 [[redirects]]
   from = "/*"
   to = "/effection/:splat"
+
+[[redirects]]
+  force = true
+  from = "/effection/api/*"
+  status = 301
```

Comment:
Why is it that we use 200 elsewhere, but 301 here?

Reply:
oh oops, that's supposed to be 200